src/device.js

```diff
@@ -62,6 +62,10 @@ export function isGoogleSearchApp(ua? : string = getUserAgent()) : boolean {
     return (/\bGSA\b/).test(ua);
 }
 
+export function isEpicGamesLauncher(ua? : string = getUserAgent()) : boolean {
```

Comment:
I think this might be a premature optimization but I wonder if we might consider doing a list that is `disallowedUserAgents = ['EpicGamesLauncher']` and loop over that in a utility for `supportsPopups`. I am mostly thinking through ways to simplify some of these conditionals for in the future as `supportsPopup` is a large set of predicates so we could trim it down with a `doesUASupportPopups()` function.

Reply:
Just to be super clear I don't think this change needs to be made now to accept the PR by any means. Just opening a conversation point.

Reply:
I believe that's the purpose of `supportsPopups()` that are being extended in line 201.

We can generalize the User Agent checks, but the function reads really well at the current size. I agree that we might need it someday as the list of agents grows, but for now this solution seems to be fine.